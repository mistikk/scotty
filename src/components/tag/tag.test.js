import React from 'react';
import renderer from 'react-test-renderer';

import Tag from './tag.view';

describe('Tag Component', () => {
  it('> render the component', () => {
    const value = 'scotty';
    const component = renderer.create(<Tag value={value} />);
    expect(component).toMatchSnapshot();
  });
});
