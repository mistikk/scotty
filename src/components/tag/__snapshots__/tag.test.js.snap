// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tag Component > render the component 1`] = `
<View>
  <Text>
    scotty
  </Text>
</View>
`;
